Automattic/wp-calypso
# Theme Discovery: Change Collection from query param to URL path

We were having some issues with SSR theme pages with query params. Since all the filtering is already done as part of the path it makes sense to include the collection view indicator in the URL path.


## Proposed Changes

* Changed the `collection` view discriminator from being a query param to part of the URL path.

## Testing Instructions



* Live link will not work since the moment you add the `?flags=themes/discovery-lots` the page won't get SSRed
* Checkout the branch locally
* Execute: `export DEBUG="calypso:server-render"` on your terminal
* then yarn and yarn start.
* Open a collection view page
* You should see the SSR logs.
* Inspect the page and check for the noindex meta and other metas.

## Pre-merge Checklist
